src/constants.rs

```diff
@@ -6,7 +6,7 @@ pub const FUELS_VERSION_FILE: &str = "fuels_version";
 
 pub const CHANNEL_LATEST_URL: &str =
     "https://raw.githubusercontent.com/FuelLabs/fuelup/gh-pages/channel-fuel-latest.toml";
-pub const CHANNEL_LATEST_FILE_NAME: &str = "channel-fuel-latest.toml";
+pub const CHANNEL_LATEST_FILE_NAME: &str = "channel-fuel-beta-4.toml";
```

Comment:
Will fuelup update in this case just download any updates to beta4 (such as patch releases)?

Reply:
Yeah update checks the channel file, since channel file name is now beta-4, it should update correctly.